src/Service.Tests/Authorization/AuthorizationHelpers.cs

```diff
@@ -69,7 +69,8 @@ public static AuthorizationResolver InitAuthorizationResolver(RuntimeConfig runt
             HashSet<string>? excludedCols = null,
             string? databasePolicy = null,
             string? requestPolicy = null,
-            string authProvider = "AppService"
+            string authProvider = "AppService",
+            DatabaseType dbType = DatabaseType.mssql
```

Comment:
For consistency across tests written for each database flavor, wouldn't we want to keep this param not defaulted to MsSql? require each test to provide db flavor param value.

Reply:
I think we'll do just fine. Actually, this parameter wasn't even present. And for every test, this method was by default creating the config for MsSql. Adding this parameter keeps the existing functionality and additionally allows us to configure the db type (if we want to).